src/cordova/plugin/android/resources/strings/values-ar/strings.xml

```diff
@@ -1,9 +1,99 @@
-<?xml version='1.0' encoding='utf-8'?>
+<?xml version="1.0"?>
 <resources>
-	<string name="app_name">Outline</string>
```

Comment:
I think before we only had the strings used by the Android code.

Is it possible to read values/strings.xml and only output those?

I worry that we are making a num_languages * num_messages increase to the app size. Any idea how much bigger the app becomes?